src/streams/EditStreamCard.js

```diff
@@ -2,9 +2,20 @@
 import React, { PureComponent } from 'react';
 import { StyleSheet, View } from 'react-native';
 
-import { Input, Label, OptionRow, ZulipButton } from '../common';
+import type { Dispatch } from '../types';
+import { connect } from '../react-redux';
+import { Input, Label, OptionRow, ZulipButton, SelectionList } from '../common';
 import styles from '../styles';
 import { IconPrivate } from '../common/Icons';
+import { getIsAdmin } from '../selectors';
+
+type StreamPostPolicy = $ReadOnly<{| key: number, value: string |}>;
+
+const STREAM_POST_POLICIES: StreamPostPolicy[] = [
+  { key: 1, value: 'All stream members' },
+  { key: 2, value: 'Only organization administrators' },
+  { key: 3, value: 'Only organization full members' },
+];
```

Comment:
The keys here are part of the server API, right? I.e. this won't work right if we used some other numbers.

It's important to draw that connection explicitly. The most natural way to do that is:
* Put the definitions of these constants -- giving names for 1, 2, and 3 -- in the types in `src/api/`.
* Then from there link to the relevant bit of API docs, which is here: https://zulipchat.com/api/add-subscriptions

Concretely, try this in `modelTypes.js`:
```js
// See stream_post_policy at https://zulipchat.com/api/add-subscriptions
export class StreamPostPolicies {
  static everyone: 1 = 1;
  static admins: 2 = 2;
  static restrict_new_members: 3 = 3;
}

export type StreamPostPolicy =
  | typeof StreamPostPolicies.everyone
  | typeof StreamPostPolicies.admins
  | typeof StreamPostPolicies.restrict_new_members;

export type Stream = {|
  // ...
  stream_post_policy: StreamPostPolicy,
```

That's modeled on what we have at `EventTypes` in `eventTypes.js`. There may be other ways to do it but that's one reasonably-clean way.

Then anyplace you would say something like `1`, say instead `StreamPostPolicies.everyone`.

Reply:
Created `streamPostPolicy.js`